about.html

```diff
@@ -0,0 +1,167 @@
+            <a href="#past-activities" onclick="closeNav()">Past Activities</a>
+            <a href="#contact" onclick="closeNav()">Contact</a>
+          </div>
+        </div>
+        <div class="pryNavBar">
+            <a href="./index.html" class="main-logo"
+              ><img
+                src="./images/logo.png" alt="main logo" class="main-logo-img" /></a>
+            <ul class="navbar-ul">
+              <li><a href="#about.html" class="desktop-nav-link">About</a></li>
+              <li>
+                <a href="#main-program" class="desktop-nav-link">Main Program</a>
+              </li>
+              <li><a href="#speakers" class="desktop-nav-link">Speakers</a></li>
+              <li><a href="#partners" class="desktop-nav-link">Partners</a></li>
+              <li><a href="#" class="desktop-nav-link">Join the Program</a></li>
+              <li><a href="#contact" class="desktop-nav-link">Contact</a></li>
+            </ul>
+          </div>
+  
+      </nav>
+      <div class="container">
+        <img
+          src="./utils/images/—Pngtree—line-decorative-design-pattern_3634817.png"
+          alt="Background image"
+          class="bg-header-image"
+        />
+        <img
+          src="./utils/images/—Pngtree—line-decorative-design-pattern_3634817.png"
+          alt="Background image"
+          class="bg-header-image-rotate"
+        />
+        <div class="pryContainerContent">
+            <div class="pryContainerContent-headers">
+              <h2 class="myColor">"Hello! Sharing world"</h2>
+              <h1 class="myColor">
+                AIOU ENTREPRENEURSHIP & TECHNOLOGY CONFERENCE
+              </h1>
+              <h3 class="year special-color heading-big">2022</h3>
+            </div>
+            <div class="headerContents text-center">
+              <p>
+                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Et,
+                illo! Rem cupiditate nemo minima voluptate laboriosam deserunt,
+                dignissimos quia incidunt id pariatur odit laborum sunt
+                reprehenderit labore blanditiis temporibus modi.
+              </p>
+            </div>
+            <div class="info">
+              <h3 class="date light-black heading-medium">
+                2022.07.19(Tue) ~ 21(Thu)
+              </h3>
+              <p class="location light-black heading-small">
+                @Conference Hall, Main Campus, AIOU Islamabad
+              </p>
+            </div>
+          </div>
+        </div>
+      </header>
+    <section class="about-us flx-col-center text-center" id="about-us">
+      <h2 class="about-us-head light-black heading-medium margin-top-big">
+        About Us
+      </h2>
+      <hr class="myLine text-center" />
+      <img src="./images/conference_photo.jpg" alt="about us community photo" class="family-photo" />
+      <div class="about-us-content">
+        <p class="about-us-paragraph">

```

Comment:
- [ ] Kindly here update the image path, so you can solve this issue.
- [ ] Kindly the image for this section is too big, kindly make it smaller and center it.
- [ ] Kindly the text is above the pic not under it.

Kindly fix these issues to match the layout of the design.

Your design
![image](https://user-images.githubusercontent.com/35535130/175190597-0762e8e3-b40a-433f-9aab-6266a6d848b4.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/175190683-713eabd2-41c4-440b-8fec-e53b23b9364d.png)